packages/helpers/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@gympass/yoga-helpers",
+  "version": "0.0.1",
+  "description": "Gympass helpers",
+  "keywords": [
+    "styled-components",
+    "helper",
+    "media",
+    "query"
+  ],
+  "author": "ggdaltoso <ggdaltoso@gmail.com>",
+  "homepage": "https://gympass.github.io/yoga/",
+  "license": "MIT",
+  "main": "src/index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
```

```suggestion
  "publishConfig": {
    "access": "public",
    "directory": "dist"
  },
```

We need to add the `directory` field to tell npm to publish the right bundle